examples/04_pong/main.rs

```diff
@@ -100,10 +110,14 @@ impl<'a, 'b, T> ECSBundle<'a, 'b, T> for PongBundle {
         &self,
         builder: ApplicationBuilder<'a, 'b, T>,
     ) -> Result<ApplicationBuilder<'a, 'b, T>> {
+        use saveload::marker::{U64Marker, U64MarkerAllocator};
         Ok(
             builder
+                .with_resource(U64MarkerAllocator::new())
+                .with_resource(Saved::default())
                 .with_resource(Score::new())
                 .with_resource(Time::default())
+                .register::<U64Marker>()
```

Comment:
These could probably do with a bundle.

Reply:
Idk. They are tied to save/load systems. And those are tied to pong bundle

Reply:
Oh!  Perhaps we ought to add that marker implementation to Amethyst itself.  U64 seems pretty generic.

Reply:
It is there

Reply:
Hm, so why can't a bundle for this be added to Amethyst if the marker structures are part of Amethyst?

Reply:
Because anyway you will include it in bundle with systems using it.

Reply:
Can one bundle add another?

Reply:
Yep, just use `ApplicationBuilder::with_bundle` from inside the bundle.